Documentation/Usage.md

```diff
@@ -250,22 +287,16 @@ let jpegData = jpeg_common_struct()
 print(jpegData)
 ```
 
-Install JPEG library using a system packager, e.g, `$ brew install jpeg`
-
-Create a directory called `CJPEG` next to the `example` directory and
-initialize it as a package that builds a system module:
-
-    example$ cd ..
-    $ mkdir CJPEG
-    $ cd CJPEG
-    CJPEG$ swift package init --type empty
+Install the JPEG library, on macOS you can use Homebrew package manager: `brew install jpeg`.
+`jpeg` is a keg-only formula, meaning it won't be linked to `/usr/local/lib`,
+and you'll have to link it manually at build time.
 
-This creates `Package.swift` file in the directory.
-Create a `module.modulemap` file so it consists of the following:
+Just like in the previous example, run `mkdir Sources/CJPEG` and add the
+following `module.modulemap`:
 
     module CJPEG [system] {
         header "shim.h"
-        header "/usr/local/include/jpeglib.h"
+        header "/usr/local/opt/jpeg/include/jpeglib.h"
```

Comment:
Should we recommend something with a VFS example?  That would make the majority of the code more cross-platform friendly.

Reply:
For this and all your other comments, thank you! I'll catch up and clean things up a bit later today (PST) and ping you when it's all updated.